nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkComponent.java

```diff
@@ -34,16 +33,15 @@
  *
  * @since 10.2
  */
-/*
- * BulkAdminService is implemented by the component and not as a service like BulkService because StreamBulkScroller
- * needs bulk configuration during its initialization. Its initialization happens during StreamService's start step
```

Comment:
I liked a brief description

Reply:
This comment was moved to `getAdapter` method for clarity.